.nvmrc

```diff
@@ -0,0 +1 @@
+10
```

Comment:
The LTS version of the node is at 12, so I think the most appropriate here is to change it to 12

Reply:
Agree. I wasn't sure about the targeted version as in package.json states `Node >=8`